libs/designsystem/item/src/item.component.scss

```
@@ -103,6 +103,11 @@
   --background: #{interaction-state.get-state-color('white', 'xxxs')};
 }
 
+// Prevents the item from being clicked when disabled
+:host(.disabled, [disabled]) {
```

I don't think you can use a selector list within a `:host()` pseudoclass (it only works for now because Angular transpiles it) - the correct syntax would be:
```suggestion
:host(.disabled),
:host([disabled]) {
```